### PR TITLE
Fixes display issues with longer dates

### LINKED ISSUE #6536: Slightly cut off text in Logbook and History dropdowns
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

In the Logbook and also History page on the start/end time dropdowns at the top, the final character in AM/PM gets slightly cut off. This below screencap is from my desktop monitor with tons of free vertical space.

image

## Environment

- Home Assistant release with the issue: 0.114.0b0
- Last working Home Assistant release (if known): I don't remember seeing this on 0.112 (new Logbook stuff)
- Browser and browser version: Chrome desktop
- Operating system: macOS

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #6536
- This PR is related to issue:
- Link to documentation pull request:

image

image

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
We should not do this, the scroller is virtual, meaning you don't have a lot of room for the actual data like this, and the height if the scroller is no longer correct.

**Comment by bramkragten** [comment]
Only thing we can do is make the inputs wider when there is room for it, and you could hide the icon on smaller screens to give more room to the inputs... 🤷‍♂️

File: src/components/ha-date-range-picker.ts [hunk 1]
```diff
@@ -179,12 +172,31 @@ export class HaDateRangePicker extends LitElement {
 
       paper-input {
         display: inline-block;
-        max-width: 200px;
+        max-width: 250px;
+        min-width: 200px;
+        --paper-input-container-shared-input-style_-_font-size: 14px;
```
Comment (bramkragten) [inline src/components/ha-date-range-picker.ts:177]: Let's only make the font smaller on smaller screens

Comment (ludeeus) [reply src/components/ha-date-range-picker.ts:177]: Don't we use 14 everywhere else anyway?

Comment (bramkragten) [reply src/components/ha-date-range-picker.ts:177]: Then it should not be specified I guess?

Comment (ludeeus) [reply src/components/ha-date-range-picker.ts:177]: We use 14 (mostly) the default in this element is 16

Comment (bramkragten) [reply src/components/ha-date-range-picker.ts:177]: Where is that default coming from? I dont we think we set `paper-input-container-shared-input-style_-_font-size` in other places?

Comment (ludeeus) [reply src/components/ha-date-range-picker.ts:177]: It originates here <https://github.com/PolymerElements/paper-styles/blob/master/classes/typography.js#L117>

Comment (bramkragten) [reply src/components/ha-date-range-picker.ts:177]: That's subhead? We don't use that here do we?

Comment (bramkragten) [reply src/components/ha-date-range-picker.ts:177]: I just checked, all our inputs are 16px, our body font size is 14px

**Review by bramkragten — APPROVED** [review]

**Pull request merged**